### ISSUE TITLE
SQL Injection Getting Bypassed

### ISSUE THREAD

**Issue by imsatender** [baidu/openrasp#208]
Hi,

I have configured an application from https://github.com/ethicalhack3r/DVWA/archive/master.zip for generating the SQL Injection using sqlmap(https://github.com/sqlmapproject/sqlmap) command as mentioned below:

`python sqlmap.py -u "http://local.dvwa.com/vulnerabilities/sqli/?id=*&Submit=Submit" --cookie "security=low; PHPSESSID=<SESSION_ID>" --dump-all -v 3`

After that, I am able to get all the database dump of performance schema and other tables.

Regards,
Satender K.

**Comment by CaledoniaProject** [comment]
We don't block attack by default, you need to enable it in official.js. I expect tons of logs in rasp/log/alarms folder

**Comment by imsatender** [comment]
> We don't block attack by default, you need to enable it in official.js. I'm expecting tons of logs in rasp/log/alarms folder with sqlmap.

Hi @CaledoniaProject,

I am getting a lot of queries blocked but still, SQLMAP is able to fetch all the database dump.

I have kept everything in block mode and information_schema also I have enabled.

Attached my official.js and screenshot from the SQLMAP output for your reference.
[official.zip](https://github.com/baidu/openrasp/files/4202779/official.zip)




 
![rasp](https://user-images.githubusercontent.com/4352869/74503521-2150b180-4f17-11ea-90e2-abfe397ae7e9.png)
